en/datatypes/float.adoc

```diff
@@ -54,12 +58,12 @@ Single quote `'` is used as a place value seperator for long integers, and can a
 
 == Comparisons
 
-All comparators can be applied on floats: `=, ==, <>, >, <, >=, &lt;=, =?`. In addition, `min`, and `max` are also supported.
+All comparators can be applied on `float!`: `=, ==, <>, >, <, >=, &lt;=, =?`. In addition, `min`, and `max` are also supported.
 
 
 == Conversion
 
-A float value can be converted at runtime to `integer!` or `string!` by using the `to` action. 
+A float value can be converted at runtime to `integer!` or `string!` by using a `to` conversion. 
```

Comment:
Let's find Peter's conversion matrix (or mine), and figure out general wording. We want to be precise, but also general. That is, listing each type supported means more edits and chances of things getting out of sync. But it will help people to know that a conversion is not supported, by design.

`-` A float value can be converted at runtime to `integer!` or `string!` by using a `to` conversion. 

`+` Floats support the following `to` conversions: `[integer! string! tag! block! ...]`